packages/react/src/components/OTPInput.tsx

```diff
@@ -36,15 +36,15 @@ export function OTPInput(props: {
             data-error={props.isInvalid}
             ref={(e) => (boxEls.current[i] = e)}
             key={i}
-            value={otp[i] === undefined ? "" : otp[i]}
+            value={otp[i] ?? ""}
```

Comment:
Prefer using nullish coalescing operator (`??`) instead of a ternary expression, as it is simpler to read.